src/couch/src/couch_db_updater.erl

```diff
@@ -294,12 +304,12 @@ collect_updates(GroupedDocsAcc, ClientsAcc, ReplicatedChanges) ->
         % local docs. It's easier to just avoid multiple _local doc
         % updaters than deal with their possible conflicts, and local docs
         % writes are relatively rare. Can be optmized later if really needed.
-        {update_docs, Client, GroupedDocs, [], ReplicatedChanges} ->
```

Comment:
do we need a clause with the original message format for internode compatibility during upgrade?

Reply:
https://github.com/apache/couchdb/pull/4673#discussion_r1296850568